tempest/run-tempest.sh

```diff
@@ -57,6 +59,16 @@ export NOSE_WITH_XUNIT=1
 export NOSE_XUNIT_FILE=tempest_xunit.xml
 
 cd /usr/share/openstack-tempest-juno/
+
+# Quick check using Javelin
+if javelin_check_resources; then
```

Comment:
I would rather:
* deploy Spinal Stack I.1.3.0 and run Sanity, with Javelin.
* Create a file in /opt/tempest-scripts (or whatever) with something like:
  `JAVELIN=I.1.3.0` inside.
* Run the upgrade to J.1.0.0 and run Sanity with Javelin in "check" mode if the previous file exits. If not, it's because we are installing a new plate-form so we will create the file with  `JAVELIN=J.1.0.0`.
 Also this file will be updated after each upgrade.

What do you think?